funnel/assets/js/util.js

```diff
@@ -186,8 +186,13 @@ export const Utils = {
       if (response.status === 500) {
         errorMsg =
           'An internal server error occurred. Our support team has been notified and will investigate.';
+      } else if (
+        response.status === 422 &&
+        response.responseJSON.error === 'requires_sudo'
+      ) {
+        window.location.href = `${window.Hasgeek.config.accountSudo}?next=${window.location.href}`;
```

Comment:
Will `next` be properly escaped? It needs to include an encoded fragment identifier so that the dialog can be reloaded on return.